provider/testdata/scripts/src/bin/download-testdata-sources.rs

```diff
@@ -12,6 +12,8 @@ use std::{io::Read, path::PathBuf};
 use tokio::{fs, io::AsyncWriteExt};
 use zip::ZipArchive;
 
+include!("../../globs.rs.data");
```

Comment:
issue: this probably won't work on crates.io: the file needs to be inside the scripts crate, since it's a subfolder

Reply:
Suggestion: leave it as doc(hidden) in the testdata crate, as it is currently

Reply:
The point is to not publish the scripts crate on crates.io.

Reply:
And to also not publish the glob definitions as part of the testdata crate, because they're implementation detail of this script.

Reply:
It's only used to download the testdata sources. (test)datagen just works with what's there.

Reply:
I meant that you should leave all the metadata, including the globs, as exports on `icu_testdata`, but move the testdata-download-sources bin into its own crate as you're doing. Then you can avoid the `..` import.

It may also be useful to install this tool locally via `cargo install .`, so it would be nice if the crate is hermetic

Reply:
I don't entirely agree that the globs are only for the download script. icu_testdata exports [icu_testdata](https://icu4x.unicode.org/doc/icu_testdata/index.html)::[paths](https://icu4x.unicode.org/doc/icu_testdata/paths/index.html)::[cldr_json_root](https://icu4x.unicode.org/doc/icu_testdata/paths/fn.cldr_json_root.html#), and these globs determine what content is in that directory.

Reply:
`cldr_json_root` is a valid directory without knowledge of the globs.

Reply:
If someone wanted to know the contents of the directory without walking the directory, or if they wanted to write their own script to populate the directory, the globs could be useful. I'm just trying to make an argument that there's a valid mental model to keep these in icu_testdata. But there's also a valid argument to move them into icu_testdata_scripts. One nice thing about that is that we can watch the entire subtree `/provider/testdata/scripts` to determine when we need to re-run `cargo make testdata-download-sources`.

Reply:
> If someone wanted to know the contents of the directory without walking the directory,

The globs won't help, some things fail to download so they don't appear in the directory.

> if they wanted to write their own script to populate the directory

Who would want to do that why? The only reason we don't just check in the zip file is because it's big, for any other purpose one can just use datagen.

Reply:
I think both Manish and I likely misunderstood that `globs.toml` is indeed in the `icu_testdata_scripts` crate; the `../..` goes from `src/bin` to the top level of that crate. It doesn't go into the parent `icu_testdata` crate.

Reply:
ah!